### DISCUSSION TITLE
fatal error при попытке полного csv экспорта

### DISCUSSION THREAD

**Discussion by dmbrtlt** [totumonline/totum-ru-issues-and-discussions#604] [ОШБК!]
<br />
<b>Fatal error</b>:  Uncaught Error: Call to a member function format() on false in /home/totum/totum-mit/totum/tableTypes/traits/WebInterfaceTrait.php:645
Stack trace:
#0 /home/totum/totum-mit/totum/tableTypes/traits/WebInterfaceTrait.php(791): totum\tableTypes\aTable-&gt;totum\tableTypes\traits\{closure}()
#1 /home/totum/totum-mit/totum/tableTypes/traits/WebInterfaceTrait.php(96): totum\tableTypes\aTable-&gt;getCsvArray()
#2 /home/totum/totum-mit/totum/moduls/Table/ReadTableActions.php(105): totum\tableTypes\aTable-&gt;csvExport()
#3 /home/totum/totum-mit/totum/moduls/Table/TableController.php(130): totum\moduls\Table\ReadTableActions-&gt;csvExport()
#4 /home/totum/totum-mit/totum/common/controllers/Controller.php(47): totum\moduls\Table\TableController-&gt;actionAjaxActions()
#5 /home/totum/totum-mit/totum/moduls/Table/TableController.php(548): totum\common\controllers\Controller-&gt;__actionRun()
#6 /home/totum/totum-mit/totum/moduls/Table/TableController.php(1185): totum\moduls\Table\TableController-&gt;__actionRun()
#7 /home/totum/totum-mit/totum/moduls/Table/TableController.php(514): totum\moduls\Table\TableController-&gt;__run()
#8 /home/totum/totum-mit/http/index.php(83): totum\moduls\Table\TableController-&gt;doIt()
#9 {main}
  thrown in <b>/home/totum/totum-mit/totum/tableTypes/traits/WebInterfaceTrait.php</b> on line <b>645</b><br />

**Comment by ttm-alexeypolunin** [comment]
Это какая-нибудь таблица с panel-view?

**Reply by dmbrtlt** [reply]
Любая таблица

**Reply by ttm-alexeypolunin** [reply]
да, надо поправить
